json/src/lib.rs

```diff
@@ -2087,6 +2087,50 @@ pub enum PubKeyOrAddress<'a> {
     PubKey(&'a PublicKey),
 }
 
+#[derive(Serialize, Clone, PartialEq, Eq, Debug)]
+/// Start a scan of the block filter index for an [output descriptor](https://github.com/bitcoin/bitcoin/blob/master/doc/descriptors.md).
+pub struct ScanBlocksRequest<'a> {
+    /// List of descriptors to scan
+    pub scanobjects: &'a [ScanBlocksRequestDescriptor],
```

Comment:
Maybe scanobjects can be a `Vec<ScanBlocksRequestDescriptor>` (since ScanBlocksRequestDescriptor already owns its descriptor), then we could do away with the lifetime `'a`.

Can possibly also derive `Default` for this whole struct.

Reply:
Pretty much just copied this from another json array example in the file https://github.com/rust-bitcoin/rust-bitcoincore-rpc/blob/23927f3e8b3bd413e0bf8b7c42b9531345bdd1a3/json/src/lib.rs#L1190

Is this giving you issues and did you test it your way? It doesn't really matter to me one way or the other.

Reply:
If you want to make a PR into my branch with how you're doing it I'm happy to merge it into my branch

Reply:
no, no issue. I think I tried it both ways.